en_US/LC_MESSAGES/django.po

```diff
@@ -10210,12 +10210,12 @@ msgstr "Start"
 #: kitsune/wiki/jinja2/wiki/document.html:95
 #, fuzzy
 msgid "Share this article: {link}"
-msgstr "Was this article helpful?"
+msgstr "Share this article: {link}"
```

Comment:
Why do we need to have `msgstr` in English? This PR will probably fix the specific problem but there are other places where the `msgstr` and the `msgid` have diverted. I am not really sure I understand the need to have msgstr for English